docs/api.md

```diff
@@ -350,6 +350,66 @@ class ResetPasswordPage extends React.Component {
 }
 ```
 
+#### ChangePasswordForm
+
+Renders a change password form. The parameter `spToken` is required in order for the token to be validated.
+
+```html
+<ChangePasswordForm spToken={this.props.location.query.sptoken} />
+```
+
+Customize the form by providing your own markup.
+
+```html
+<ChangePasswordForm>
+  <div spIf="form.sent">
+    <p>Your new password has been set. Please <LoginLink />.</p>
+  </div>
+  <div spIf="!form.sent">
+    <p>
+      <label htmlFor="password">Password</label><br />
+      <input id="password" type="password" name="password" required />
+    </p>
+    <p>
+      <label htmlFor="confirmPassword">Password (again)</label><br />
+      <input id="confirmPassword" type="password" name="confirmPassword" required />
```

Comment:
Creating a custom form without the `required` attributes will make it possible to submit the password without filling in the `confirmPassword` field.